apps/web/src/actions/documents/getContentByPath.ts

```diff
@@ -14,9 +15,13 @@ export const getDocumentContentByPathAction = withProject
     }),
     { type: 'json' },
   )
-  .handler(async ({ input }) => {
+  .handler(async ({ input, ctx }) => {
+    const commitsScope = new CommitsRepository(ctx.project.workspaceId)
+    const commit = await commitsScope
+      .getCommitById(input.commitId)
+      .then((r) => r.unwrap())
```

Comment:
Why do we need to use the `workspaceId` to get the `commitScope`? Shouldn't we use the `commitId` instead?

Reply:
nein, commitId alone is not scoped by tenant, you have to first scope by tenant (the workspace) before you can do any  search of commits